expression/builtin_info.go

```diff
@@ -803,6 +803,9 @@ func (b *builtinTiDBDecodePlanSig) evalString(row chunk.Row) (string, bool, erro
 		return "", isNull, err
 	}
 	planTree, err := plancodec.DecodePlan(planString)
+	if err != nil {
+		return planString, false, nil
+	}
 	return planTree, false, err
```

Comment:
err --> nil ?

Reply:
If the `planString` can't be decoded, we should return planString directly, and it should not be considered as error.

Reply:
I mean the err in line must be nil. So we can `return nil` here

Reply:
Do you mean return `planTree, false, nil` in L809?

Reply:
Yes